common/migrations/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Package for creating admin account.
+
+When you run project with pgsqlDB, admin model
```

Comment:
```
This will automatically add pre-configured admin account
to your db during migration. Specify admin's email via
ADMIN_EMAIL environment variable. The password is
hardcoded to 12345678, don't forget to reset it.
```